tools/importers/configs/SynchronizeTypographyTokensConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "destinationPath": "lib/styles/settings/typography/",
+  "bins": [
+    {
+      "id": "633d96230e6a79321e1dec41",
+      "name": "wnl",
+      "isTheme": false,
+      "files": {
+        "variablesRaw": {
+          "destinationVariables": "_variables.scss",
+          "destinationVariablesCss": "_variables-css.scss",
+          "destinationVariablesCssJson": "_variables-css.json"
+        },
+        "tokens": {
+          "destination": "_tokens.scss",
+          "destinationJson": "_tokens.json"
+        }
+      }
+    }
+  ],
+  "jsonBinApiUrl": "https://api.jsonbin.io/v3/b/",
+  "xMasterKey": "$2b$10$pGYu2VSnUSqm/6m2kMaJSOwc39KdqiTppeIv/qFcAaaqkRm.J/akW"
```

Comment:
Will move it to .env

Reply:
Fixed.

Reply:
IMO, we have to rotate it. You can access git history i public repos

Reply:
I changed the key.

Reply:
does UX use the same key? Do we need to inform them that the old one isn't valid anymore?

Reply:
@kvas-damian problem solved. After consultation with UX I created 2 separated `X-ACCESS-KEY`:
1. Figma - with full permissions - UX is testing it in Figma.
2. Importers - only for reading.
So we are not using `X-MASTER-KEY`, I also updated passwd.